Add contact form section

Ruthmy opened the pull request:
- Create the new section for the contact form.
- Set the CSS style for the contact form section.
- Setted the ACTION and POST attributes for the form with `Formspree` integration.

Ruthmy committed:
Added the contact form section with an skeleton of the form

Booze33 committed:
Added the contact form

Ruthmy committed:
Added CSS styling of the contact form

Ruthmy committed:
Fixed Stylelint errors about eol-whitespace

Ruthmy committed:
Fixed Stylelint errors about block-opening-brace-space-before

IngDavidTM commented on index.html:
```diff
@@ -167,49 +167,100 @@ <h3 class="recent-works__title">Multi-Post Stories Gain+Glory</h3>
+				<label for="email">
+					<input type="text" name="email" id="email" autocomplete="email" placeholder="Email address" class="contact-form__input" required>
+				</label>
+				<label for="textarea">
```
- [ ] While you did a good job using `textarea` for the text field :+1:, I think it would be better if you add a `placeholder='Enter text here'` attribute instead of using a value 'Enter text here' because it would improve the User experience

IngDavidTM commented on index.html:
```diff
@@ -167,49 +167,100 @@ <h3 class="recent-works__title">Multi-Post Stories Gain+Glory</h3>
+					<input type="text" name="name" id="name" autocomplete="name" placeholder="Full Name" maxlength="30" class="contact-form__input" required>
+				</label>
+				<label for="email">
+					<input type="text" name="email" id="email" autocomplete="email" placeholder="Email address" class="contact-form__input" required>
```
- [ ] Good job creating the `input` element for email :+1:, but as a project requirement the email input's type should be equals email.

IngDavidTM commented on index.html:
```diff
@@ -167,49 +167,100 @@ <h3 class="recent-works__title">Multi-Post Stories Gain+Glory</h3>
+				<label for="textarea">
+					<textarea name="message" minlength="10" maxlength="500" required>Enter text here</textarea>
+				</label>
+                <center><button type="submit">Get in touch</button></center>			    
```
- [ ] You did a fantastic job styling the form of your portfolio :+1:, but as the project requires the button interactions such as hover, active (when it's pressed), disabled, and enable (default state) should be styled as shown in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=IPAfT7AoceJlYmE2-0).

![Screenshot from 2023-02-21 23-12-00](https://user-images.githubusercontent.com/105117832/220519985-822c8524-062a-49d9-b9ed-f7afcbe9fbb2.png)

IngDavidTM reviewed (changes requested):
## Hi @team 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media2.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif"/>

## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Ruthmy committed:
Fixed HTML and CSS details
- Set a placeholder attribute instead of a value for the textarea.
- Fixed the input type for email input.
- Set the button interactions for the hover and active events.
- Set the background image for the form section.
- Set the resizer style for the textarea element.

Ruthmy committed:
Fixed Stylelint errors about Unexpected whitespace

Ruthmy committed:
Fixed Stylelint errors about descending-specificity

topeogunleye reviewed (approved):
### Hi Team 👋, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
![congratulations](https://user-images.githubusercontent.com/52934128/163691476-087b4b9c-825b-40a3-85a1-d1bde44ffa53.gif)
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- PR has a good description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
